boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/util/IAMConfiguration.java

```diff
@@ -48,6 +48,8 @@
 
     public static final String READY_MARKER = "iam.ready.marker";
 
+    public static final String ERROR_MARKER = "iam.error.markercle";
```

Comment:
Is it normal that the string ends with "cle"? I would have expected "iam.error.markercle".